package.json

```diff
@@ -1,13 +1,16 @@
 {
-	"name": "o-errors",
-	"private": true,
+	"description": "A decoupled events-based mechanism for components to report errors and an API for products to report client-side errors.",
+	"keywords": ["error", "errors"],
+	"name": "@financial-times/o-errors",
+	"dependencies": {
+		"raven-js": "^3.14.0"
+	},
 	"devDependencies": {
 		"eslint": "^7.8.1",
 		"eslint-config-origami-component": "^2.1.0",
-		"origami-ci-tools": "^2.0.2",
 		"remark": "^13.0.0",
 		"remark-lint": "^8.0.0",
-		"remark-preset-lint-origami-component": "^1.1.3",
+		"remark-preset-lint-origami-component": "^2.0.0-beta.1",
```

Comment:
I was going to suggest using `prerelease` here instead of a specific version but the package doesn't have the `prerelease` dist-tag